assets/src/components/routeLadder.tsx

```diff
@@ -241,6 +248,12 @@ const RouteLadder = ({
             inTestGroup(TestGroups.DetoursPilot) &&
             inTestGroup(TestGroups.DetourRouteSelection)
           }
+          onClickAddDetour={
+            onAddDetour &&
+            (() => {
+              onAddDetour(route)
+            })
```

Comment:
suggestion(non-blocking): since we're not changing the display based on the presensce of this variable, we could simplify this callback, but of course is just taste and is optional.

But maybe I'm missing something about the function being passed in conditionally?
```suggestion
            (() => {
              onAddDetour?.(route)
            })
```